style.css

```
@@ -781,3 +781,172 @@ p {
 .input1 > label {
   display: none;
 }
+
+/* Mobile Popup */
+
+.popUp {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 24px;
+  position: fixed;
+  width: 343px;
+  height: 750px;
+  left: calc(50% - 343px / 2);
+  top: 30px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+}
+
```

[OPTIONAL] The problem can be realized depending on the size of the laptop screen used to test your page. Please work with the suggestions below.

```suggestion

.popUp {
  box-sizing: border-box;
  display: flex;
  flex-direction: column;
  align-items: center;
  padding: 16px;
  gap: 10px;
  position: fixed;
  width: 343px;
  left: calc(50% - 343px / 2);
  top: 10px;
  bottom: 10px;
  background: #fff;
  border: 1px solid #dfe1e6;
}

```
Consider reducing the value of both `top` and `gap` properties, and then adding one for `bottom` in your fixed positioning. For example:

```JavaScript
gap: 10px;
top: 10px;
bottom: 10px;
```
Then, remove the height property.

I have found the above steps to be able to solve the issue the CRs are raising.